webrecorder/browsertrix
# Show org storage quotas in dashboard

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/1198



### Changes

- Displays storage quota in subdivided meter
- Updates icon colors
- Adds new `<btrix-meter>` component

### Manual testing

1. Log in as superadmin
2. Go to org without quotas. Verify metrics are shown with additional "Data Stored" stat
3. Go to org with quota. Verify meter is shown with hoverable value bars for each storage category

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Dashboard - No quota | <img width="365" alt="Screenshot 2023-09-25 at 2 11 24 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/33dc1492-c657-4a25-92ff-ab107d8eed62"> |
| Dashboard - Quota reached | <img width="369" alt="Screenshot 2023-09-25 at 2 12 06 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/13952a16-95e4-4e10-a814-5e249c1d1736"> |
| Dashboard - Storage available | <img width="371" alt="Screenshot 2023-09-25 at 2 11 32 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/3c05333d-8bbd-498b-ba5e-5605d2b5ad55"> |
| Dashboard - Storage available | **Subcategory tooltip:**<br /><img width="450" alt="Screenshot 2023-09-25 at 2 02 43 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/c0f9c15c-4091-453b-b55b-87daa9733d74"><br /><br />**Subcategory tooltip w/ less than 1%:**<br /><img width="437" alt="Screenshot 2023-09-25 at 2 02 50 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/af6bd85d-6895-4bcf-88e3-ef33d724ffea"><br /><br />**Available space tooltip:**<br /><img width="444" alt="Screenshot 2023-09-25 at 2 02 54 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/0d2a255b-159e-4639-ae11-2ad49f266542"> |
